examples/IRServer/IRServer.ino

```diff
@@ -148,5 +150,6 @@ void setup(void) {
 }
 
 void loop(void) {
+  mdns.update();
```

Comment:
Also Travis fails with the following when compiling for ESP32:
```
/home/travis/build/crankyoldgit/IRremoteESP8266/examples/IRServer/IRServer.ino:153:8: error: 'class MDNSResponder' has no member named 'update'
   mdns.update();
```

Reply:
I added a conditional.